content/cloud/cloud-command-reference.md

```diff
@@ -28,6 +30,8 @@ url = "https://github.com/fermyon/developer/blob/main/content/cloud/cloud-comman
 
 ## Spin Cloud Command
 
+> TODO: I added headers above for new CLI commands (spin cloud sqlite create, spin cloud link), but their descriptions need to be added below
```

Comment:
@kate-goldenring - would be great to get your help here if you have the time 🙏

Reply:
Added the link commands and `sqlite create`